sdk/formrecognizer/Azure.AI.FormRecognizer/perf/Program.cs

```diff
@@ -0,0 +1,7 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Reflection;
+using Azure.Test.Perf;
+
+await PerfProgram.Main(Assembly.GetEntryAssembly(), args);
```

Comment:
Nothing to see here. This is the Main function we literally copied from the Perf project template.